namui/namui-type/src/types/units/rect.rs

```diff
@@ -625,3 +625,40 @@ where
         }
     }
 }
+
+impl<T> Rect<T>
+where
+    T: Debug + rkyv::Archive,
+    <T as rkyv::Archive>::Archived: Debug,
```

Comment:
`T: Debug + rkyv::Archive` is required for map?

Reply:
that;s very right I thought it is required